feat(upgrade project): Upgrade project to newest jsass and refactor t…

beo4 opened the pull request:
…o new maven dependecies

 - jsass from 5.7.3 to 5.10.3
 - upgrade to Junit5
 - upgrade to new maven plugins

beo4 committed:
feat(upgrade projekt): Upgrade project to newest jsass nad refactor to new maven dependecies

 - jsass from 5.7.3 to 5.10.3
 - upgrade to Junit5
 - upgrade to new maven plugins

beo4 committed:
feat(travis): use openjdk to compile as no oraclejdk8 is available anymore in travis

 - use openjdk to compile as no oraclejdk8 is available anymore in travis

beo4 commented:
Hi @mickleroy,

i have updated the Project to use AEM uber.jar 6.4.6, also renewed the maven-plugins and junit. maybe you are still maintaining this :)

beo4 committed:
fix(bnd-compile): added dependent jars into bundle jar

mickleroy commented:
Thanks @beo4. 

I'm not really maintaining this project anymore. It served it's purpose but there's better ways to work with Sass and other front end technologies nowadays.

I had a quick glance at the change and there seemed to be lot's of unnecessary Maven dependencies added (and a BMW reference?) that need to be cleaned up. I'll take a closer look when I get some more time.

beo4 committed:
chore(dependency): removed unused dependecies from pom

 - add maven-dependency-plugin to identify unused dependencies
 - removed all dependenceis
 - added ignoredDependencies to maven-dependency-plugin, otherwise build is not running anymore

beo4 commented:
Hi @mickleroy,

removed the BMW reference, was a c&p issue.  

We were the opinion using webpack, etc is the modern way. As Adobe is promoting the in-server compilation with core components (the Grid is less only) we testing this now with your sass compiler, as sass is our company preferred precompiler.
